ci/cron/src/Main.hs

```diff
@@ -90,24 +92,43 @@ http_get url = do
       _ -> Exit.die $ unlines ["GET \"" <> url <> "\" returned status code " <> show status <> ".",
                                show $ HTTP.responseBody response]
 
-newtype Version = Version (Int, Int, Int, Maybe String)
+data StringOrInt = I Int | S String
```

Comment:
Can we just use the `semver` library which we use in other places instead of trying to implement our own semver type and comparison?

Reply:
What I needed here was pretty simple and I knew how to implement it. It's also pretty short, and generally speaking I dislike relying on libraries when the equivalent code is short and simple.

Also, I can't figure out how to use the SemVer library from its documentation, and I did not care to learn about lenses just for this. Happy to give you a chance to change my mind if you want to volunteer some time to explain them to me.

Reply:
Very happy to explain them to you. Let’s leave it out of this PR and then whenever you’re interested in, we can take a look at switching it over to `semver` together.